.github/workflows/discussion-triage.yml

```
@@ -0,0 +1,28 @@
+name: Discussion Triage
+on:
```

You can define [`run-name`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#run-name) and leverage the `github` context to help distinguish the workflow run from others:

```suggestion
name: Discussion Triage
run-name: ${{ github.event.issue.title }}
on:
```

---

Neat!

---

Applied.